File: adopt-a-pup/adoption-service/src/main/java/com/redhat/do328/adoptApup/adoptionservice/model/AdoptionApplication.java
package com.redhat.do328.adoptApup.adoptionservice.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class AdoptionApplication {
    private String username;
    private String animalId;
    private Residency residency;
    private int squareFootageOfHome;
    private boolean kidsUnder16;
    private String occupation;
    private boolean ownOtherAnimals;
    private String email;
}

File: adopt-a-pup/web-app/src/Models/AdoptionApplication.ts
import { Residency } from "./Residency";

export interface AdoptionApplication {
    username: string;
    animalId: string;
    residency: Residency;
    squareFootageOfHome: number;
    kidsUnder16: boolean;
    occupation: string;
    ownOtherAnimals: boolean;
    email: string;
}


